src/Files/MultilingualResources/Files.zh-Hant.xlf

```diff
@@ -3456,47 +3456,47 @@ Files 不會收集、存儲、共享或發布任何使用者的個資。
         </trans-unit>
         <trans-unit id="Extract" translate="yes" xml:space="preserve">
           <source>Extract</source>
-          <target state="new">Extract</target>
+          <target state="new">解壓縮</target>
```

Comment:
These translations won't show up unless the state is changed to `translated`. Can you update them?